packages/backend-output-schemas/src/storage/v1.ts

```diff
@@ -1,9 +1,12 @@
 import { z } from 'zod';
 
+const bucketSchema = z.record(z.string(), z.string());
```

Comment:
should we type this stronger as well?

Reply:
Done d67854ec809441ddd20bc88407be944aa6950d42